### ISSUE TITLE
If possible, "delete artist account" shouldn't appear under the merch form

### ISSUE THREAD

**Issue by LouisLouiseKay** [funmusicplace/mirlo#859]
It's misleading as this can appear to be the delete button for the item if you're not paying attention closely.

**Comment by simonv3** [comment]
Yeah I kind of think this button should be moved. Maybe it's in the "edit artist details" form?
